README.md

```diff
@@ -327,8 +327,16 @@ msg = topic.publish "new-message"
 # Retrieve a subscription
 sub = pubsub.subscription "my-topic-sub"
 
-# Pull available messages
-msgs = sub.pull
+# Create a subscriber to listen for available messages
+subscriber = sub.listen do |msg|
+  # process msg
+  msg.ack!
+end
+
+subscriber.start
```

Comment:
Could you add a comment that this method will start a background thread that will call the block passed to the listen method?

Reply:
Yes, will do.